docker/Dockerfile

```diff
@@ -18,17 +18,11 @@ FROM ubuntu:16.04
 RUN apt-get update
 RUN apt-get install -y openssh-server gcc make sudo cmake apt-transport-https software-properties-common binutils g++
 
-# 2. Oracle Java Jdk 8
-RUN \
-  echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && \
-  add-apt-repository -y ppa:webupd8team/java && \
-  apt-get update && \
-  apt-get install -y oracle-java8-installer && \
-  rm -rf /var/lib/apt/lists/* && \
-  rm -rf /var/cache/oracle-jdk7-installer
+# 2. OpenJdk 8
+RUN sudo apt-get install -y openjdk-8-jdk
```

Comment:
This will probably fail without adding the right repository.
The version that works for me is:
```dockerfile
RUN add-apt-repository -y ppa:webupd8team/java && \
    apt-get update && \
    sudo apt-get install -y openjdk-8-jdk
```